### DISCUSSION TITLE
Splitting worksheet?

### DISCUSSION THREAD

**Discussion by boulepick** [exceljs/exceljs#2442] [Q&A]
Hi There,

is there a way to split a worksheet into multiple smaller worksheets so that work can be performed to this worksheets. i'm trying to use the the "excel-as-json" module to convert excel to json but running into memory issues for excel sheets that have more than 100000 rows.

**Comment by atljoseph** [comment]
Why can i not do addWorksheet(workSheet)?? 
Why is this not built that way??
Why do you have to add a new blank Worksheet, then copy all the data manually??
Very basic ...

**Comment by Siemienik** [comment]
https://github.com/exceljs/exceljs/issues/770
